src/main/kotlin/no/nav/syfo/cronjob/statusendring/PublishDialogmoteStatusEndringService.kt

```diff
@@ -32,13 +32,46 @@ class PublishDialogmoteStatusEndringService(
             pDialogmote = database.getDialogmote(id = moteId).first(),
             personIdent = database.getMoteDeltakerArbeidstaker(moteId).personIdent,
             virksomhetsnummer = database.getMoteDeltakerArbeidsgiver(moteId).virksomhetsnummer,
+            isStatusEndretForMotedeltakerBehandler = isRelatedToMotedeltakerBehandler(
+                dialogmoteStatusEndret = dialogmoteStatusEndret,
+            ),
         )
         dialogmoteStatusEndringProducer.sendDialogmoteStatusEndring(kDialogmoteStatusEndring)
 
         database.updateMoteStatusEndretPublishedAt(
             moteStatusEndretId = dialogmoteStatusEndret.id,
         )
     }
+
+    fun isRelatedToMotedeltakerBehandler(
```

Comment:
Tanken min her er at i stedet for å sett  `sykmelder=true` for alle DialogmoteStatusEndret hvor behandler var med i første innkalling(og visa versa for innkalling uten behandler), så er dette er forsøk på å heller sette sykmelder=true om behandler var relevant for en gitt DialogmoteStatusEndret.

F.eks at sykmeldter=true om behandler ble innkalt, og behandler=false om ferdigstilling ikke gjaldt behandler.

Da skal dette i teorien kunne vise om en valgfri deltaker faktisk er knyttet til en gitt endring og ikke bare om en deltaker er knyttet innkallingen.

Mulig det er bedre måter å gjør dette på, f.eks lagre i database om endringen gjaldt en deltaker.
Også mulig at koden ikke fungerer som tenkt her nå, har ikke testet dette.

Tar gjerne i mot tilbakemelding på dette :)

Reply:
Tror dette kan funke omtrent slik du har gjort det 👍